JavaScript/JavaScriptSDK/JavaScriptSDK.csproj

```diff
@@ -56,6 +56,15 @@
     <TypeScriptRemoveComments>false</TypeScriptRemoveComments>
     <TypeScriptSourceMap>true</TypeScriptSourceMap>
     <TypeScriptModuleKind>AMD</TypeScriptModuleKind>
+    <TypeScriptJSXEmit>None</TypeScriptJSXEmit>
+    <TypeScriptCompileOnSaveEnabled>True</TypeScriptCompileOnSaveEnabled>
+    <TypeScriptNoImplicitAny>False</TypeScriptNoImplicitAny>
+    <TypeScriptOutFile>min\$(OutFileBaseName).js</TypeScriptOutFile>
+    <TypeScriptOutDir />
+    <TypeScriptNoEmitOnError>True</TypeScriptNoEmitOnError>
+    <TypeScriptMapRoot>
+    </TypeScriptMapRoot>
+    <TypeScriptSourceRoot>https://raw.githubusercontent.com/Microsoft/ApplicationInsights-JS/master/JavaScript</TypeScriptSourceRoot>
```

Comment:
You're right. I thought about it and I was planning to make two branches: develop and master (the same as we have for other SDKs), but this will still not work if somebody is using an older version of the SDK. Adding a release tag to the URL is a better solution. 

I'll add the release tag to the URL and update the release script to do that automatically - #443.